src/lexedata/exporter/cognates.py

```diff
@@ -353,17 +355,22 @@ def get_segments(self, form):
     )
     parser.add_argument(
         "--metadata",
-        help="Path to metadata file for dataset input",
+        nargs="?",
+        type=Path,
         default="Wordlist-metadata.json",
+        help="Path to the metadata.json. The metadata file describes the dataset. Default: ./Wordlist-metadata.json",
     )
     parser.add_argument(
-        "--excel", help="Excel output file path", default="Cognates.xlsx"
+        "--output-file",
```

Comment:
The excel output is not optional (in particular, you can't sensibly write Excel to stdout), so it should be a positional argument, not an optional switch.